app/gcl/agwa/section6/page.module.css

```
@@ -0,0 +1,550 @@
+.agwa-section6-root {
+    background: var(--color-guide-background);
+    color: var(--color-guide-foreground);
+    font-family: 'Hiragino Sans', 'Noto Sans JP', 'Yu Gothic UI', 'Meiryo', system-ui, sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Meiryo" (font-family-name-quotes)

(font-family-name-quotes)